timed/employment/migrations/0005_merge_20170810_0924.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-08-10 07:24
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('employment', '0003_auto_20170808_1403'),
+        ('employment', '0004_auto_20170809_1647'),
+    ]
+
+    operations = [
+    ]
```

Comment:
I think we should try to keep the migrations clean. There seems to have been a conflict and I think it is better instead of a merging migration to delete the 3 migrations in this PR (0003, 0004 and 0005) and run makemigrations again.